docs/current_docs/adopting.mdx

```diff
@@ -4,13 +4,121 @@ slug: /adopting
 
 # Adopting Dagger
 
-You’ve done the [quickstart](quickstart/index.mdx), and now you’re probably asking yourself, "What should I do next?" 
+Once you have [completed the quickstart](quickstart), and learned the basic concepts, it's time to take the next step: adopting Dagger in your project - what we call *daggerizing*. But how does one Daggerize a project, exactly?
 
-We get it, looking at that many lines of YAML can be intimidating. 
-Since each pipeline is unique in tooling and functionally, our general answer is to "start simple". Pick a part of your pipeline to Daggerize first, and then continue to Daggerize from there.
+Every project is different, and every pipeline is a unique snowflake (usually that's not a good thing). So we can't give you an exact step-by-step guide of how to daggerize your project. But we can share the collective experience of the Dagger community, distilled into a simple framework.
 
-Not sure which part of your pipeline to Daggerize first? Reach out to us on [Discord](https://discord.gg/dagger-io) via our [help forum](https://discord.com/channels/707636530424053791/1030538312508776540). Our team and your fellow Daggernauts will be happy to provide more specific guidance on your use case there! 
+## Phase 1: research
 
-Want to see how others have used Dagger? Check out our [case studies](https://dagger.io/resources/Case-Studies) or watch [demos](https://dagger.io/resources/Videos) from community members like yourself. 
+### Join our Discord!
 
-Do you have a use case that you’d like to talk about privately? Reach out to us at solutions@dagger.io
+Before anything, drop what you're doing and join our [Discord server](https://discord.gg/dagger-io). It is the beating heart of the Dagger community, and one of the killer features of the platform. It is the best place to ask questions, get inspiration, and ask for help when you are stuck.
+
+Consider yourself warned: adopting Dagger without joining our Discord is certainly possible, but you probably won't be as productive, and it definitely won't be as fun!
+
+### Look for inspiration
```

Comment:
We should curate this a bit more, there are many case studies and videos that predate modules, so folks finding those will lead to confusion. 

Perhaps a good place to highlight some of the mature DtW projects? 

If we do link out to more generic indexes, we should provide a pointer letting people know to limit their exploration to things that were published recently.

Reply:
Makes sense. I'll also leave that for a follow-up PR (trying to get this one merged before 0.12, so we benefit from the influx of new visitors drawn by the announcement)